docs/placeholder.md

```diff
@@ -0,0 +1,208 @@
+# Placeholder
+
+[![Maven Central](https://img.shields.io/maven-central/v/com.google.accompanist/accompanist-placeholder)](https://search.maven.org/search?q=g:com.google.accompanist)
+
+A library which provides a [modifier][modifier] for display 'placeholder' UI while content is loading.
+
+More information on the UX provided by this library can be found on the Material Theming [Placeholder UI](https://material.io/design/communication/launch-screen.html#placeholder-ui) guidelines.
+
+There are actually two versions of the library available:
+
+* *Placeholder Foundation*: Provides the base functionality and depends on Jetpack Compose Foundation. This version requires the app to provide all of the colors to display.
```

Comment:
What about re-ordering these as I expect most to use the materail version and only rarely drop down to foundation?

Reply:
Hmmm, it's hard to explain the material version before explaining the foundation version. We explicitly call out below that most apps should use the Material. Still think it's useful?

I could make the para below more prominent instead?

Reply:
Went with this:

<img width="716" alt="Screen Shot 2021-06-10 at 12 48 42" src="https://user-images.githubusercontent.com/227486/121520010-4dd0ec80-c9ea-11eb-92ab-4b3b6b9e6abc.png">

Reply:
Works for me.